js_modules/dagit/packages/core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -112,31 +111,32 @@ export const AssetGraphExplorer: React.FC<Props> = (props) => {
   );
 };
 
-export const AssetGraphExplorerWithData: React.FC<
-  {
-    allAssetKeys: AssetKey[];
-    assetGraphData: GraphData;
-    graphQueryItems: GraphQueryItem[];
-    liveDataByNode: LiveData;
-    liveDataRefreshState: QueryRefreshState;
-    applyingEmptyDefault: boolean;
-  } & Props
-> = (props) => {
-  const {
-    options,
-    setOptions,
-    explorerPath,
-    onChangeExplorerPath,
-    onNavigateToSourceAssetNode: onNavigateToSourceAssetNode,
-    liveDataRefreshState,
-    liveDataByNode,
-    assetGraphData,
-    graphQueryItems,
-    applyingEmptyDefault,
-    fetchOptions,
-  } = props;
-
+type WithDataProps = {
+  allAssetKeys: AssetKey[];
+  assetGraphData: GraphData;
+  graphQueryItems: AssetGraphQueryItem[];
+  liveDataByNode: LiveData;
+  liveDataRefreshState: QueryRefreshState;
+  applyingEmptyDefault: boolean;
+} & Props;
```

Comment:
Is this more idiomatic for props?

```
interface WithDataProps extends Props {
  // etc.
}
```

Reply:
Hmm I think it's more consistent with what we do elsewhere, I'll change it in a follow-up 👍  From a technical standpoint I'm actually not sure what the differences are... 😳